script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -472,12 +506,12 @@ Please, for God's sake, leave now. Jump away in one big leap, just like you did
 [sha:72f156af21122a4281dbafe67d25ab2db5fa1f59]{
 -- Page 391, Offset 15750.
 -- 「なに、この家の人？
-"What, is this someone from the house?
+"What, is this someone who lives here?
 }
 [sha:f69c89b3c2ce2e1333ea016447da6318fe0de9f6]{
 -- Page 391, Offset 15751.
 -- いい機会だし、挨拶していこうかなー？」
-As good an opportunity as any――maybe I'll go greet them."
+This is as good an opportunity as any――maybe I'll go say hi."
```

Comment:
Nothing wrong with dropping some subjects now and then.
This is colloquial speech.

Reply:
I agree about colloq speech, but I don't think this is one I'd actually use

Reply:
I feel like it become abnormally strict this way, given how Arcueid says it.

Maybe you could replace it

> Maybe I'll go say hi――might as well.

Or something to that effect.